dashboard/config/regional_partners_school_districts.tsv

```diff
@@ -11550,62 +11550,119 @@ Southern Nevada Regional Professional Development Program	3200480
 Southern Nevada Regional Professional Development Program	3200510
 Southern Nevada Regional Professional Development Program	5900061
 Southern Nevada Regional Professional Development Program	5900153
+Code/Interactive	3600001	csp	August 14 - 18, 2017
 Code/Interactive	3600001
```

Comment:
No - this is by design.

Column 1 is the regional partner, column 2 is the district ID, column 3 and 4 are optional columns saying "for this course, the workshop date is X"